crime-applications-adaptor/src/main/java/uk/gov/justice/laa/crime/applications/adaptor/mapper/crimeapply/PassportedMapper.java

```diff
@@ -1,35 +1,50 @@
 package uk.gov.justice.laa.crime.applications.adaptor.mapper.crimeapply;
 
 import uk.gov.justice.laa.crime.applications.adaptor.model.crimeapplicationsadaptor.common.Passported;
+import uk.gov.justice.laa.crime.applications.adaptor.model.criminalapplicationsdatastore.Applicant;
 import uk.gov.justice.laa.crime.applications.adaptor.model.criminalapplicationsdatastore.MaatApplicationExternal;
 
 import javax.validation.constraints.NotNull;
+import java.util.Objects;
 
 class PassportedMapper {
 
-    private static final boolean INCOME_SUPPORT_DEFAULT_FALSE = false;
-    private static final boolean STATE_PENSION_DEFAULT_FALSE = false;
-    private static final boolean JOB_SEEKER_DEFAULT_FALSE = false;
-    private static final boolean EMPLOYMENT_SUPPORT_DEFAULT_FALSE = false;
-    private static final boolean CLAIMED_BY_PARTNER_DEFAULT_FALSE = false;
+    private static final boolean FALSE = false;
+    private static final boolean TRUE = true;
     private static final Passported.WhoDwpChecked WHO_DWP_CHECKED_DEFAULT_NULL = null;
 
     @NotNull
     Passported map(MaatApplicationExternal crimeApplyMaatApplicationExternal) {
 
         Passported passported = new Passported();
 
-        passported.setBenefitIncomeSupport(INCOME_SUPPORT_DEFAULT_FALSE);
-        passported.setBenefitGuaranteedStatePension(STATE_PENSION_DEFAULT_FALSE);
-        passported.setJobSeeker(JOB_SEEKER_DEFAULT_FALSE);
-        passported.setBenefitEmploymentSupport(EMPLOYMENT_SUPPORT_DEFAULT_FALSE);
-        passported.setBenefitClaimedByPartner(CLAIMED_BY_PARTNER_DEFAULT_FALSE);
+        passported.setBenefitIncomeSupport(FALSE);
+        passported.setBenefitGuaranteedStatePension(FALSE);
+        passported.setBenefitJobSeeker(FALSE);
+        passported.setBenefitUniversalCredit(FALSE);
+        passported.setBenefitEmploymentSupport(FALSE);
+        passported.setBenefitClaimedByPartner(FALSE);
         passported.setWhoDwpChecked(WHO_DWP_CHECKED_DEFAULT_NULL);
 
-        if(!crimeApplyMaatApplicationExternal.getMeansPassport().isEmpty()){
+        if (!crimeApplyMaatApplicationExternal.getMeansPassport().isEmpty()) {
             passported.setMeansPassport(String.valueOf(crimeApplyMaatApplicationExternal.getMeansPassport().get(0)));
         }
 
+        if (Objects.nonNull(crimeApplyMaatApplicationExternal.getClientDetails().getApplicant()) &&
```

Comment:
We can you reduce the duplication here by extracting
`crimeApplyMaatApplicationExternal.getClientDetails().getApplicant()`
into a local variable